guides/_subsections/re-upload-a-recording-to-archives.md

```diff
@@ -8,40 +8,43 @@ previously retrieved recording in a new Cryostat instance.
 <ol>
   <li>
     {% include howto_step.html
-      summary="Select an application"
-      image-name="select-target-application.png"
-      text="
-        Although the archived storage is global to the Cryostat instance and is
-        not target application specific, the view for <i>Archived Recordings</i>
-        is only visible when a target application is selected.
+      summary="Navigate to the <i>Archives</i> tab"
+      image-name="2.2.0/re-upload-a-recording-to-archives-1.png"
+      caption="Select the Uploads tab."
+      text="The entire Archives view contains several tabs related to performing operations
+      on archived recordings. See <a href='#view-archived-recordings'>Viewing Archived Recordings</a>.
       "
     %}
   </li>
-  <li>
-    {% include_relative _subsections/common/navigate-to-recordings.md %}
-  </li>
   <li>
     {% include howto_step.html
-      summary="Select the Archived Recordings tab"
-      image-name="re-upload-a-recording-to-archives-1.png"
+      summary="Select the Recording to Upload"
+      image-name="2.2.0/re-upload-a-recording-to-archives-2.png"
+      caption="
+        Click the '+' button to bring up the upload prompt. Then click <i>Browse...</i> and select the
+        <code>.jfr</code> file to upload. Note that the file must follow the Cryostat archive recording
+        naming convention. Mouse over the <sup>[?]</sup> tooltip on the prompt for more information.
+      "
     %}
   </li>
   <li>
     {% include howto_step.html
-      summary="Select the Recording to Upload"
-      image-name="re-upload-a-recording-to-archives-2.png"
+      summary="<i>(Optional)</i> Add metadata labels"
+      image-name="2.2.0/re-upload-a-recording-to-archives-3.png"
       caption="
-        Click the '+' button, then click <i>Browse...</i> and select the .jfr
-        file to upload. Then click 'Submit'.
+        Click <i>Show metadata options</i> to add optional metadata labels to recording.
+        You can either add labels by clicking the <i>Add Label</i> button, or by uploading
+        a custom <code>.json</code> labels file from your local storage. For more on Cryostat
```

Comment:
Should we add a sample of this `.json` file or mention it is the same as the one saved when downloading a recording?

Reply:
I think just mentioning that it's the same one that would be downloaded is good. It's somewhat subject to change between versions, and I don't expect users will all that often be manually editing these files anyway. Much more likely to just select the accompanying metadata download to restore it.